### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by satyajit321**
Missing # causing the internal reference link to be broken for "determine-if-centrally-deployed" leading to 404 not found error when clicked.

Corrected link:
https://github.com/OfficeDev/office-js/tree/release/add-in-ids#determine-if-centrally-deployed-add-ins-are-using-legacy-tokens

404 - page not found
The release branch of office-js does not contain the path  add-in-ids/determine-if-centrally-deployed-add-ins-are-using-legacy-tokens.

### CODE REVIEW

**Review by lindalu-MSFT — CHANGES_REQUESTED** [review]
I think we only need ONE link at the end of this section. We don't need it at the end of every sentence.

"For next steps, see [Determine if centrally deployed add-ins are using legacy tokens](#determine-if-centrally-deployed-add-ins-are-using-legacy-tokens)."

**Comment by lindalu-MSFT** [comment]
@satyajit321

**Comment by satyajit321** [comment]
I'm fine with additional edits, I have only updated the hyperlinks issue. If content needs modification it can be done directly or seperately. I'm not sure what's next to be done here.

File: add-in-ids/README.md [hunk 1]
```diff
@@ -33,15 +33,15 @@ Run the following command to get a list of all add-ins deployed at the organizat
 
 The command lists all add-ins you deployed on your tenant. If any add-ins have a value for the **AssetID** then they are deployed from the Microsoft store and you can search the `add-ins-using-exchange-tokens.xlsx` spreadsheet to see if they use legacy tokens. For next steps, see [Determine if Microsoft store add-ins are using legacy tokens](#determine-if-microsoft-store-add-ins-are-using-legacy-tokens).
 
-Add-ins without a value for the **AssetID** are centrally deployed. These can’t be found in the spreadsheet. For next steps, see [Determine if centrally deployed add-ins are using legacy tokens](determine-if-centrally-deployed-add-ins-are-using-legacy-tokens).
+Add-ins without a value for the **AssetID** are centrally deployed. These can’t be found in the spreadsheet. For next steps, see [Determine if centrally deployed add-ins are using legacy tokens](#determine-if-centrally-deployed-add-ins-are-using-legacy-tokens).
```
Comment (lindalu-MSFT) [inline add-in-ids/README.md]:
```suggestion
Add-ins without a value for the **AssetID** are centrally deployed. These can’t be found in the spreadsheet. 
```

File: add-in-ids/README.md [hunk 2]
```diff
@@ -33,15 +33,15 @@ Run the following command to get a list of all add-ins deployed at the organizat
 
 The command lists all add-ins you deployed on your tenant. If any add-ins have a value for the **AssetID** then they are deployed from the Microsoft store and you can search the `add-ins-using-exchange-tokens.xlsx` spreadsheet to see if they use legacy tokens. For next steps, see [Determine if Microsoft store add-ins are using legacy tokens](#determine-if-microsoft-store-add-ins-are-using-legacy-tokens).
 
-Add-ins without a value for the **AssetID** are centrally deployed. These can’t be found in the spreadsheet. For next steps, see [Determine if centrally deployed add-ins are using legacy tokens](determine-if-centrally-deployed-add-ins-are-using-legacy-tokens).
+Add-ins without a value for the **AssetID** are centrally deployed. These can’t be found in the spreadsheet.
```
Comment (lindalu-MSFT) [inline add-in-ids/README.md]:
```suggestion
Add-ins without a value for the **AssetID** are centrally deployed. These can’t be found in the spreadsheet. 

For next steps, see [Determine if centrally deployed add-ins are using legacy tokens](#determine-if-centrally-deployed-add-ins-are-using-legacy-tokens).
```

File: add-in-ids/README.md [hunk 3]
```diff
@@ -33,15 +33,17 @@ Run the following command to get a list of all add-ins deployed at the organizat
 
 The command lists all add-ins you deployed on your tenant. If any add-ins have a value for the **AssetID** then they are deployed from the Microsoft store and you can search the `add-ins-using-exchange-tokens.xlsx` spreadsheet to see if they use legacy tokens. For next steps, see [Determine if Microsoft store add-ins are using legacy tokens](#determine-if-microsoft-store-add-ins-are-using-legacy-tokens).
```
Comment (lindalu-MSFT) [inline add-in-ids/README.md]:
```suggestion
The command lists all add-ins you deployed on your tenant. If any add-ins have a value for the **AssetID** then they are deployed from the Microsoft store and you can search the `add-ins-using-exchange-tokens.xlsx` spreadsheet to see if they use legacy tokens. 
```

File: add-in-ids/README.md [hunk 4]
```diff
@@ -57,7 +59,7 @@ Open the `add-in-list-export.csv` file in Excel. If the list doesn't open with t
 
 The **MarketplaceAssetID** column will list an asset ID if the add-in was deployed from the Microsoft store. You can search the spreadsheet to see if these are using legacy tokens. For next steps, see [Determine if Microsoft store add-ins are using legacy tokens](#determine-if-microsoft-store-add-ins-are-using-legacy-tokens).
```
Comment (lindalu-MSFT) [inline add-in-ids/README.md]:
```suggestion
The **MarketplaceAssetID** column will list an asset ID if the add-in was deployed from the Microsoft store. You can search the spreadsheet to see if these are using legacy tokens. 
```

File: add-in-ids/README.md [hunk 5]
```diff
@@ -55,9 +57,9 @@ The command exports an `add-in-list-export.csv` file with a list of all add-ins
...
+The **MarketplaceAssetID** column will list an asset ID if the add-in was deployed from the Microsoft store. You can search the spreadsheet to see if these are using legacy tokens. 
 
-Add-ins that don't have a **MarketplaceAssetID** were deployed by the user from the organization or by sideloading. These aren’t listed in the spreadsheet and are either line-of-business add-ins or 3rd party add-ins. For next steps, see [Determine if centrally deployed add-ins are using legacy tokens](determine-if-centrally-deployed-add-ins-are-using-legacy-tokens).
+Add-ins that don't have a **MarketplaceAssetID** were deployed by the user from the organization or by sideloading. These aren’t listed in the spreadsheet and are either line-of-business add-ins or 3rd party add-ins. For next steps, see [Determine if centrally deployed add-ins are using legacy tokens](#determine-if-centrally-deployed-add-ins-are-using-legacy-tokens).
```
Comment (lindalu-MSFT) [inline add-in-ids/README.md]:
```suggestion
Add-ins that don't have a **MarketplaceAssetID** were deployed by the user from the organization or by sideloading. These aren’t listed in the spreadsheet and are either line-of-business add-ins or 3rd party add-ins. For next steps, see [Determine if centrally deployed add-ins are using legacy tokens](#determine-if-centrally-deployed-add-ins-are-using-legacy-tokens).

For next steps, see [Determine if centrally deployed add-ins are using legacy tokens](#determine-if-centrally-deployed-add-ins-are-using-legacy-tokens).
```

File: add-in-ids/README.md [hunk 6]
```diff
@@ -31,17 +31,19 @@ Run the following command to get a list of all add-ins deployed at the organizat
...
 
 Next, run the following command in PowerShell to get a list of organization add-ins. This list may include additional add-ins not found from the previous command.
 
 - `Get-App -OrganizationApp | ft DisplayName, MarketplaceAssetID, AppId, ProviderName`
 
 The command lists all organization add-ins deployed on your tenant. The **MarketplaceAssetID** lists the asset ID of any add-ins deployed from the Microsoft store. You can search the spreadsheet to see if these are using legacy tokens. For next steps, see [Determine if Microsoft store add-ins are using legacy tokens](#determine-if-microsoft-store-add-ins-are-using-legacy-tokens).
 
-Add-ins without a value for the **MarketplaceAssetID** are centrally deployed. These can’t be found in the spreadsheet. For next steps, see [Determine if centrally deployed add-ins are using legacy tokens](determine-if-centrally-deployed-add-ins-are-using-legacy-tokens).
+Add-ins without a value for the **MarketplaceAssetID** are centrally deployed. These can’t be found in the spreadsheet. For next steps, see [Determine if centrally deployed add-ins are using legacy tokens](#determine-if-centrally-deployed-add-ins-are-using-legacy-tokens).
```
Comment (lindalu-MSFT) [inline add-in-ids/README.md]:
```suggestion
Add-ins without a value for the **MarketplaceAssetID** are centrally deployed. These can’t be found in the spreadsheet. 
```

File: add-in-ids/README.md [hunk 7]
```diff
@@ -31,17 +31,19 @@ Run the following command to get a list of all add-ins deployed at the organizat
...
-Add-ins without a value for the **AssetID** are centrally deployed. These can’t be found in the spreadsheet. For next steps, see [Determine if centrally deployed add-ins are using legacy tokens](determine-if-centrally-deployed-add-ins-are-using-legacy-tokens).
+Add-ins without a value for the **AssetID** are centrally deployed. These can’t be found in the spreadsheet. 
+
+For next steps, see [Determine if centrally deployed add-ins are using legacy tokens](#determine-if-centrally-deployed-add-ins-are-using-legacy-tokens).
 
 Next, run the following command in PowerShell to get a list of organization add-ins. This list may include additional add-ins not found from the previous command.
 
 - `Get-App -OrganizationApp | ft DisplayName, MarketplaceAssetID, AppId, ProviderName`
 
 The command lists all organization add-ins deployed on your tenant. The **MarketplaceAssetID** lists the asset ID of any add-ins deployed from the Microsoft store. You can search the spreadsheet to see if these are using legacy tokens. For next steps, see [Determine if Microsoft store add-ins are using legacy tokens](#determine-if-microsoft-store-add-ins-are-using-legacy-tokens).
```
Comment (lindalu-MSFT) [inline add-in-ids/README.md]:
```suggestion
The command lists all organization add-ins deployed on your tenant. The **MarketplaceAssetID** lists the asset ID of any add-ins deployed from the Microsoft store. You can search the spreadsheet to see if these are using legacy tokens. 
```

File: add-in-ids/README.md [hunk 8]
```diff
@@ -31,17 +31,19 @@ Run the following command to get a list of all add-ins deployed at the organizat
...
 
 - `Get-App -OrganizationApp | ft DisplayName, MarketplaceAssetID, AppId, ProviderName`
 
-The command lists all organization add-ins deployed on your tenant. The **MarketplaceAssetID** lists the asset ID of any add-ins deployed from the Microsoft store. You can search the spreadsheet to see if these are using legacy tokens. For next steps, see [Determine if Microsoft store add-ins are using legacy tokens](#determine-if-microsoft-store-add-ins-are-using-legacy-tokens).
+The command lists all organization add-ins deployed on your tenant. The **MarketplaceAssetID** lists the asset ID of any add-ins deployed from the Microsoft store. You can search the spreadsheet to see if these are using legacy tokens. 
 
-Add-ins without a value for the **MarketplaceAssetID** are centrally deployed. These can’t be found in the spreadsheet. For next steps, see [Determine if centrally deployed add-ins are using legacy tokens](determine-if-centrally-deployed-add-ins-are-using-legacy-tokens).
+Add-ins without a value for the **MarketplaceAssetID** are centrally deployed. These can’t be found in the spreadsheet.
```
Comment (lindalu-MSFT) [inline add-in-ids/README.md:42]:
```suggestion
Add-ins without a value for the **MarketplaceAssetID** are centrally deployed. These can’t be found in the spreadsheet. 

For next steps, see [Determine if centrally deployed add-ins are using legacy tokens](#determine-if-centrally-deployed-add-ins-are-using-legacy-tokens).
```

File: add-in-ids/README.md [hunk 9]
```diff
@@ -55,9 +59,11 @@ The command exports an `add-in-list-export.csv` file with a list of all add-ins
 
 Open the `add-in-list-export.csv` file in Excel. If the list doesn't open with the correct columns, go to the **Data** tab, in the **Get & Transform Data group**, and choose **From Text/CSV**.
 
-The **MarketplaceAssetID** column will list an asset ID if the add-in was deployed from the Microsoft store. You can search the spreadsheet to see if these are using legacy tokens. For next steps, see [Determine if Microsoft store add-ins are using legacy tokens](#determine-if-microsoft-store-add-ins-are-using-legacy-tokens).
+The **MarketplaceAssetID** column will list an asset ID if the add-in was deployed from the Microsoft store. You can search the spreadsheet to see if these are using legacy tokens. 
+
+Add-ins that don't have a **MarketplaceAssetID** were deployed by the user from the organization or by sideloading. These aren’t listed in the spreadsheet and are either line-of-business add-ins or 3rd party add-ins. For next steps, see [Determine if centrally deployed add-ins are using legacy tokens](#determine-if-centrally-deployed-add-ins-are-using-legacy-tokens).
```
Comment (lindalu-MSFT) [inline add-in-ids/README.md]:
```suggestion
Add-ins that don't have a **MarketplaceAssetID** were deployed by the user from the organization or by sideloading. These aren’t listed in the spreadsheet and are either line-of-business add-ins or 3rd party add-ins. 
```

**Review by lindalu-MSFT — DISMISSED** [review]
I removed redundant sentences and added just one at the end of each section.

**Review by lindalu-MSFT — APPROVED** [review]

**Pull request merged**